### PR TITLE
Added search form on the places index page.

### PR DESCRIPTION
**Pull request by GabrielSandoval**
image

places#show has a search form but places#index doesn't have. I added a search form in the index page of the 'places' so that the users may easily search the place they want to see.

@pozorvlak Sorry I messed up my last PR. Here is a new one. I left the gitignore untouched this time.

### CODE REVIEW

**Comment by pozorvlak** [comment]
Looks good! However, it should be covered by a test. I would say "just copy the existing test for the Show page", but

1. there isn't one
2. the [existing tests for the Show page](https://github.com/Growstuff/growstuff/blob/dev/spec/views/places/show.html.haml_spec.rb) are in a deprecated style.

So you'd need to write a whole new file of "feature" tests for the places-index page. Do you feel confident to do that? The good news is that feature tests are [more straightforward to write](https://github.com/Growstuff/growstuff/blob/dev/spec/features/members_list_spec.rb) - that's why we deprecated view and controller tests!

**Comment by GabrielSandoval** [comment]
Thanks @pozorvlak! I'll start working on this as soon as I'm confident about testing. I'm at bit new to rails and I'm excited to try this one.

GabrielSandoval,
AELOGICA Intern

**Comment by GabrielSandoval** [comment]
Hi @pozorvlak,

I added the feature test for the places search form. Please see if it's all good.

Thanks!

File: config/initializers/geocoder.rb [hunk 1]
```diff
@@ -15,3 +15,16 @@
   Geocoder.configure(:lookup => :nominatim)
 end
 
+Geocoder::Lookup::Test.add_stub(
```
Comment (pozorvlak) [inline config/initializers/geocoder.rb]: Could you please move this to `config/environments/test.rb` along with the other Geocoder stubs?

Comment (GabrielSandoval) [inline config/initializers/geocoder.rb]: Hi, I moved the stubs to config/environments/test.rb. Just tell me if there's anything more needed to be done.

**Comment by pozorvlak** [comment]
Excellent - thanks!

**Comment by GabrielSandoval** [comment]
Thanks @pozorvlak!

**Pull request merged**